docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/bias_quantification.ipynb

```diff
@@ -0,0 +1,1387 @@
+{
```

Comment:
<p>As above, I think this can just be wasting -&gt; SQ-LNS -&gt; stunting with a correlation between wasting and stunting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>So I agree with your comment above that we should simplify the generic diagram to R1 -&gt; intervention -&gt; R2 with a correlation between R1 and R2. I'm thinking it might be worthwhile to include these sorts of applied examples that might appear different than that exact structure of the generic diagram but do contain that same underlying structure (in addition to potentially distracting details) to demonstrate that it might not always to be obvious in the context of an actual project. I'm thinking I'll include <em>both </em>versions here with some commentary about this. Does that sound alright to you?</p>

Reply:
Ah, good idea! I think that sounds great if you clearly mark which section of the messy, real-world example has the pattern. For example, having everything but putting the part I suggested in bold or something.